src/lib/redux/siteData.ts

```diff
@@ -1,8 +1,11 @@
 import { getAllSites } from '../airtable/request';
-import { addCustomer, saveSiteData, setCurrSite, setLoadingForSiteData } from './siteDataSlice';
+import { SiteRecord, CustomerRecord, SiteId } from '../airtable/interface';
+import { EMPTY_SITE, saveSiteData, setCurrSite, setLoadingForSiteData } from './siteDataSlice';
+import { saveCustomerData } from './customerDataSlice';
 import { store } from './store';
 
-const refreshSiteData = async (loadSilently: boolean): Promise<void> => {
+//  TODO: DON'T KNOW WHERE TO PUT THIS????? Shouldn't be here.
```

Comment:
I was thinking about moving this function to `userData`, but otherwise couldn't find a proper place for it.

Reply:
It doesn't reallyy matter, but I feel like I might just create a new file with literally one function if you want to make sure logic of where things go is consistent 😅  - maybe call it `async`?

Reply:
That's a good point. I'll probably just create the new file.

As for async: I was afraid `async` in the name might be misleading because our first login call makes it kinda synchronous to the user (the first `refreshData` passes `loadSilently=false`) so that they can't access the rest of the data until the pull succeeds. Not entirely sure what's standard though